susemanager/susemanager.changes

```diff
@@ -1,3 +1,4 @@
+- add automatic mode to mgr-create-bootstrap-repo
```

Comment:
It seems to me we need another entry to mention the support for Beta channels, no?

Reply:
Ok, as we have bugreports about it, I will add a line

Reply:
done